src/main/java/io/datasearch/diseasedata/store/dengdipipeline/ingestion/data/DiseaseData.java

```diff
@@ -0,0 +1,63 @@
+package io.datasearch.denguestore.data;
```

Comment:
package name shoud be compying with the file path
it should be like` io.datasearch.diseasedata.store.dengdipipeline.ingestion.data`
We can remove the` data` part as well. Look into this